server/src/main/scala/org/http4s/server/staticcontent/ResourceService.scala

```diff
@@ -32,6 +146,7 @@ object ResourceService {
       preferGzipped: Boolean = false)
 
   /** Make a new [[org.http4s.HttpRoutes]] that serves static files. */
+  @deprecated("use ResourceServiceBuilder", "1.0.0")
```

Comment:
This will be released in a milestone:

```suggestion
  @deprecated("use ResourceServiceBuilder", "1.0.0-M1")
```

Reply:
@rossabaker made the changes